docs/developers/guard-accounts.md

```diff
@@ -158,18 +182,26 @@ In the meantime you can use the **MultiversX Guardians Test - BETA** version:
 **How to activate it?**
 1. In Ledger Live
 2. Go to My Ledger
-3. Go to Settings
+3. Go to ⚙️
```

Comment:
optional: could be Settings (⚙️) instead of ⚙️ - maybe some browsers won't display the emoji
also for the other changes, leave the emoji as fallback near the description (Developer Mode - ✅ -> Enable ✅ Developer Mode)

Reply:
good idea